### ISSUE TITLE
[Bug Report] App crashing on startup: uiThreadHandler has not been initialized

### ISSUE THREAD

**Issue by NTB45** [chr56/Phonograph_Plus#17]
App crashing on startup

**Crash log**
If app crashed, copy the logs if available.
```
Crash Report:

Package name:    player.phonograph.plus
App version:     0.3.0-rc01 (290)
Git Commit Hash: a251ffe1c31cf911a108
Android version: 9 (SDK 28)
Device brand:    samsung  (by samsung)
Device model:    SM-G610F (code: on7xreflte)
Product name:    on7xreflteub
Build version:   PPR1.180610.011.G611MUBSACUD1 
                 (G611MUBSACUD1)
Hardware:        samsungexynos7870
Language:        en

java.lang.RuntimeException: Unable to create service player.phonograph.service.MusicService: e4.l: lateinit property uiThreadHandler has not been initialized
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:3789)
	at android.app.ActivityThread.access$1400(ActivityThread.java:240)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1822)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:214)
	at android.app.ActivityThread.main(ActivityThread.java:7124)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:494)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:975)
Caused by: e4.l: lateinit property uiThreadHandler has not been initialized
	at player.phonograph.service.MusicService.runOnUiThread(Unknown Source:10)
	at player.phonograph.appwidgets.AppWidgetBig.f(SourceFile:6)
	at k7.a.e(SourceFile:2)
	at player.phonograph.service.MusicService.e(Unknown Source:19)
	at player.phonograph.service.MusicService.onCreate(Unknown Source:15)
	at android.app.ActivityThread.handleCreateService(ActivityThread.java:3777)
	... 8 more


```

**How to reproduce**
Steps to reproduce the bugs or unexpected behaviors:
Start the app
Crash
**Screenshots (Optional)**
(if available)

**Other information**
…
Phone is rooted from magisk but I think its unrelated
App worked fine before last update
Will switch to it till it gets fixed

**Comment by chr56** [comment]
Did it crash every time?

**Comment by chr56** [comment]
Ok, I repuduce it now. Currently, try to remove all of  desktop AppWidge until I fix it please. Thanks!

**Comment by chr56** [comment]
Fix in [v0.3.0](https://github.com/chr56/Phonograph_Plus/releases/tag/v0.3.0)

**Comment by NTB45** [comment]
Thanks for the good work
